src/lib/components/Sidebar.svelte

```diff
@@ -74,6 +76,21 @@ The sidebar/drawer.
             <!-- svelte-ignore a11y-click-events-have-key-events -->
             <li><a on:click={closeDrawer}><TextAppearanceIcon />Text Appearance</a></li>
             <div class="dy-divider m-1" />
+            {#if menuItems}
+                {#each menuItems as item}
+                    <li>
+                        <a href={item.link['default']}>
+                            <img
+                                src="icons/{item.images[2].file}"
```

Comment:
You need to update to the most recent SAB (build 10-Nov 2022). The location of the menu item icons is now `icons/menu-items/` so that user-created icons don't conflict in name with icons that might be delivered with the app.